tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/buildorder/BuildOrderTest.xtend

```diff
@@ -38,18 +43,36 @@ class BuildOrderTest extends AbstractIdeTest {
 	 *            test workspace
 	 */
 	@SafeVarargs
-	 def final void test(String expectation, Pair<String, List<Pair<String, String>>>... projectsModulesContents) {
+	 def final void test(String buildOrder, Pair<String, List<Pair<String, String>>>... projectsModulesContents) {
+		test(buildOrder, emptyList, projectsModulesContents);
+	}
+
+	/**
+	 * @param expectation
+	 *            string of project names ordered in build order
+	 * @param projectsModulesContents
+	 *            test workspace
+	 */
+	@SafeVarargs
+	 def final void test(String buildOrder, Collection<Collection<String>> cycles, Pair<String, List<Pair<String, String>>>... projectsModulesContents) {
```

Comment:
odd white space: there is a space character right before `def` (same in previous method)